src/views/HomeNextStep.vue

```diff
@@ -112,87 +82,73 @@ export default class Home extends Vue {
     }
 
     &--wrapper {
-        z-index: 1;
-        margin-top: 100px;
-        margin-bottom: 100px;
         display: flex;
         flex-direction: column;
-        gap: 30px;
+        z-index: 1;
+        background-color: var(--bg-wallet-light);
+        padding: 32px;
+        border-radius: 18px;
+        gap: 1.5rem;
+        max-width: 800px;
         align-items: center;
-        .header {
-            font-size: main.$l-size;
-            font-weight: 700;
-            display: flex;
-            align-items: center;
-            gap: 15px;
-        }
-        .content {
-            max-width: 750px;
+        margin: 0px 10px;
+        box-shadow: rgba(0, 0, 0, 0.1) 0px 0px 5px !important ;
+        border: 1px solid rgba(145, 158, 171, 0.24) !important;
+    }
+
+    .content {
+        &--title {
+            font-size: 2.75rem;
+            color: var(--text-color);
             text-align: center;
-            &--desc {
-                text-align: left;
-                background-color: var(--bg-light-2);
-                border-radius: var(--border-radius-sm);
-                margin-bottom: 30px;
-                font-size: 20px;
-                padding: 25px;
-                font-weight: 500;
-            }
-            &--access-create {
-                // margin-bottom: 30px;
-                font-size: 18px;
-                color: var(--primary-color-light);
-            }
         }
-    }
-    .buttons-wrapper {
-        display: flex;
-        justify-content: center;
-        gap: 30px;
-        flex-wrap: wrap;
-        align-items: center;
-        .v-btn {
-            min-width: 175px;
+        &--desc {
+            font-size: 1.125rem;
+            line-height: 1.5;
         }
     }
-}
-.warningg {
```

Comment:
removing this class leads to have the attetion word without color, fix the color of the text for the "ATTENTION" word